### ISSUE TITLE
Blank Solution is not a project template in VS 2022

### ISSUE THREAD

**Issue by carlin-q-scott** [MicrosoftDocs/visualstudio-docs#8261]
I tried to follow the directions for creating a Blank Solution, but that option is not available.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: d7d9f53c-6ece-ea3c-9a51-9edd0efeebe3
* Version Independent ID: 9c927a8e-3ebd-0c8a-76cc-9b1c980b98a3
* Content: [Create projects & solutions - Visual Studio (Windows)](https://docs.microsoft.com/en-us/visualstudio/ide/creating-solutions-and-projects?view=vs-2022)
* Content Source: [docs/ide/creating-solutions-and-projects.md](https://github.com/MicrosoftDocs/visualstudio-docs/blob/main/docs/ide/creating-solutions-and-projects.md)
* Product: **visual-studio-windows**
* Technology: **vs-ide-general**
* GitHub Login: @TerryGLee
* Microsoft Alias: **tglee**

**Comment by TerryGLee** [comment]
@carlin-q-scott I'm sorry to hear that; unfortunately, I cannot repro your experience; I was able to follow the procedure and successfully create a blank solution, as follows:

![image](https://user-images.githubusercontent.com/23244653/178612874-b6112c3f-729c-4d6d-90b1-446b6a539ae7.png)

Are you using Visual Studio for Windows? (In other words, not Visual Studio for Mac or VS Code, per chance?)

**Comment by carlin-q-scott** [comment]
Looks like your team added it back very recently:
![devenv_eg0UIeC63Y](https://user-images.githubusercontent.com/2521644/178768103-a45f9d85-4b0d-4817-aa4b-ba7478148b83.gif)

**Comment by TerryGLee** [comment]
@carlin-q-scott Thanks for the follow up along with the handy animation! I'm happy to see that it's working again for you. 

I'm not sure about the timing of template updates and how they are managed by the engineering team; I can only report that everything worked correctly at the time our documentation team initially wrote the content and each time we updated it since. 

Going forward though, should you find an error in the product such as this, please consider reporting that via the [Report a Problem](https://docs.microsoft.com/visualstudio/ide/how-to-report-a-problem-with-visual-studio) tool that appears both in the Visual Studio Installer and in the Visual Studio IDE. The engineering team uses the data from this tool to schedule bug fixes, etc., and might be able to provide further info about what could have gone wrong, when it was fixed, etc., via that forum.

Meanwhile, we'll close this documentation issue now; we thank you for your interest in improving the Visual Studio documentation and hope to hear from you again!

**Comment by Firegarden** [comment]
I just upgraded from Version 17.2.5 to 17.2.6 and I am now seeing the blank solution option is available. Seems this is resolved for the latest version.

**Comment by davicbaba** [comment]
Same here with VS2022 Version 17.2.6

**Comment by Netislander** [comment]
Have the same problem and it's intermitten.  Sometimes it's there other times it's not.  I've even pinned it to my recent project templates.  Restarting VS doesn't work.

**Comment by carlin-q-scott** [comment]
@TerryGLee Do you know why it keeps disappearing for us? Maybe this issue needs to be moved to whatever component manages the online templates?

**Comment by TerryGLee** [comment]
@carlin-q-scott, et al ... please report this product error via the [Report a Problem](https://docs.microsoft.com/visualstudio/ide/how-to-report-a-problem-with-visual-studio) tool that appears both in the Visual Studio Installer and in the Visual Studio IDE. The engineering team uses the data from this tool to schedule bug fixes, etc., and might be able to provide further info about what could have gone wrong, when it was fixed, etc., via that forum.

**Comment by carlin-q-scott** [comment]
I have reported this issue through the Visual Studio Feedback site: https://developercommunity.visualstudio.com/t/Blank-Solution-project-template-missing/10114064?port=1025&fsid=01218593-58f8-4304-be32-c81d3561a971&entry=problem

**Comment by Raguf** [comment]
Guys I found video, where helped with this problem https://www.youtube.com/watch?v=3L2KZ6kqrec

**Comment by akshay-zz** [comment]
Just as an alternative you can use the following command to create the sln file.
`dotnet new sln --name MySolution`

**Comment by AdmiralSnyder** [comment]
@pavel-zhukov resetuserdata did not work for me. can you confirm that it really fixed it for you? because that's quite a disruptive operation you're suggesting there...

**Comment by KieranFoot** [comment]
This is still an issue. Sometimes updates seem to remove this template.

**Comment by heartbitt** [comment]
Yes. The last update Blank Slolution has been removed from my system. I never have this issue before

**Comment by svdHero** [comment]
I am having the same problem with Microsoft Visual Studio Professional 2022 (64-bit) Version 17.4.2. Never had the problem before, but right now there is no Blank Solution Template anymore.

@TerryGLee could you reopen this, please?

**Comment by CalvinQuark** [comment]
I'm using version 17.4.4. I don't see a Blank Solution template in the _Create new project_ screen with _All languages_, _All platforms_, and _All project types_ selected.

**Comment by bkrachev** [comment]
Yesterday I updated to 17.5 and created a new project from Blank Solution (I have it pinned). Today I turned on the computer, VS, new project, and... no Blank Solution. The solution is to go to the Installer, Individual Components, and search for project and Item templates. It was unchecked for me.
![Screenshot (11)](https://user-images.githubusercontent.com/22366215/221348053-5b3818d8-34b7-47aa-a680-a804f4faf534.png)

**Comment by MRobinson-dev** [comment]
I had this option until I upgraded to Windows Version 17.5.0   now there is no blank solution

**Comment by MRobinson-dev** [comment]
> bkrachev

why should a blank solution template be only in full framework?

**Comment by bkrachev** [comment]
I've no idea. But it's not the first time it happens to me.

**Comment by rhundhausen** [comment]
v17.5.0 for me and, ya, the Blank Solution template comes and goes inside the IDE.

**Comment by TerryGLee** [comment]
Everyone, please upvote or add your feedback to the official record for this product issue, here: https://developercommunity.visualstudio.com/t/Blank-Solution-project-template-missing/10114064?entry=problem. Please note that the Developer Community is the right place to get this issue fixed. Thanks.

**Comment by syazdian** [comment]
> Guys I found video, where helped with this problem [youtube.com/watch?v=3L2KZ6kqrec](https://www.youtube.com/watch?v=3L2KZ6kqrec)

Thanks, saved me

**Comment by sdg002** [comment]
> _%ProgramFiles%\Microsoft Visual Studio\ `your version` \ `your edition` \Common7\IDE\devenv /resetuserdata_ fixes it.

After much annoyance, `/resetuserdata` worked for me
![image](https://user-images.githubusercontent.com/20245330/236625051-be7d8b72-c126-4c58-bb74-3c506b551c7d.png)

![image](https://user-images.githubusercontent.com/20245330/236625078-7f049b28-cb3e-4980-82ed-79ca26f50623.png)

**Comment by TymurGubayev** [comment]
this is an issue for me, in VS 17.6.2

**Comment by davidliang2008** [comment]
I just updated to v 17.6.2 and it looks like they have fixed it.

![image](https://github.com/MicrosoftDocs/visualstudio-docs/assets/1124420/80f4394f-eba4-4dec-a8ed-ad4f26dbf513)

![image](https://github.com/MicrosoftDocs/visualstudio-docs/assets/1124420/6ad77345-8569-46eb-8175-39556d02e3b5)

**Comment by jimfoye** [comment]
Missing for me in 17.6.2.

**Comment by alimohamadi7** [comment]
Missing for me in 17.6.3 &17.6.2.

**Comment by mihben** [comment]
Missing from 17.6.4.
It can be solved by reinstall the .NET Framework project and item templates component, but very annoying to do it after every update. I have already done three times.

**Comment by ekarpov** [comment]
Still missing in 17.6.4
